packages/sources/coinmetrics/src/endpoint/realized-vol.ts

```diff
@@ -0,0 +1,62 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { config } from '../config'
+import { transport } from '../transport/realized-vol'
+
+const RESULT_PATHS = ['realVol1Day', 'realVol7Day', 'realVol30Day'] as const
```

Comment:
nice, we can use something like this to standardize

Reply:
Agree - only reason I've kept returning all in `data` and only the specified in `result` is to be consistent with Tiingo.

I'd be happy to trim this down, but we should probably do this for all the Realized Vol EAs for consistency.